framework/chi_configuration.h

```diff
@@ -1,9 +1,7 @@
-#ifndef CHI_CONFIGURATION_H
-#define CHI_CONFIGURATION_H
+#pragma once
 
-#define PROJECT_VERSION "1.0.2"
+#define PROJECT_VERSION       "1.0.2"
```

Comment:
why the extra white space?

Reply:
Must be a side effect of my regular expression.